### PR TITLE
note on cases sensitivity of endpoint names

### PR DESCRIPTION
**Pull request by tmasternak**
ping @timbussmann

### CODE REVIEW

**Comment by SimonCropp** [comment]
isnt this a bug?

**Comment by timbussmann** [comment]
> isnt this a bug?

I asked this myself too, but as this heavily depends on the transports (e.g. are queue names case-sensitive?) we would have to explicitly implement core & transport logic to be case-insensitive. I think I would classify this as a feature?

File: nservicebus/msmq/routing_content_core_[6,).partial.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ NOTE: When running under ASP.NET the `instance-mapping.xml` file may not be loca
 
 snippet: InstanceMappingFile
 
+NOTE: The endpoint names are case sensitive e.g. if endpoint name is used in routing API and the instance mapping file, the file entries will be used only when there is a case sensitive match.
```
Comment (timbussmann) [inline nservicebus/msmq/routing_content_core_[6,).partial.md]: should this be a WARN instead?

Comment (timbussmann) [inline nservicebus/msmq/routing_content_core_[6,).partial.md]: also, do we need the whole example part?

Comment (tmasternak) [inline nservicebus/msmq/routing_content_core_[6,).partial.md]: I would lean to keeping the example part. This doco [page](https://docs.particular.net/nservicebus/msmq/routing) shows usage of endpoint names only in xml file (which uses each name once) so it might be confusing for readers what is matched with what.

Comment (timbussmann) [inline nservicebus/msmq/routing_content_core_[6,).partial.md]: I don't have strong feelings about this. Maybe rephrase to something like `The endpoint names are case sensitive. Endpoint names used in the routing name and the instance mapping file require an exact string match` to avoid the `e.g.` ?
